doc/source/API/index.rst

```diff
@@ -5,20 +5,96 @@ API reference
 This section describes PyAEDT core classes, methods, and functions
 for AEDT apps and modules. Use the search feature or click links
 to view API documentation.
+The Ansys Electronics Desktop (AEDT) is a platform that enables true electronics system design.
+`AEDT <https://www.ansys.com/it-it/products/electronics>`_ provides access to the Ansys gold-standard
+electro-magnetics simulation solutions such as Ansys HFSS,
+Ansys Maxwell, Ansys Q3D Extractor, Ansys Siwave, and Ansys Icepak using electrical CAD (ECAD) and
+Mechanical CAD (MCAD) workflows.
+In addition, it also includes direct links to the complete Ansys portfolio of thermal, fluid,
+and Mechanical solvers for comprehensive multiphysics analysis.
+Tight integration among these solutions provides the user with unprecedented ease of use for setup and
+faster resolution of complex simulations for design and optimization.
+
+.. image:: ../Resources/aedt.png
+  :width: 800
+  :alt: AEDT Applications
+  :target: https://www.ansys.com/it-it/products/electronics
+
+The PyAEDT API includes classes for apps and modules. You must initialize the
+app to get access to all modules and methods. Available apps are:
```

Comment:
You must initialize AEDT....
or
You must initialize a PyAEDT app....
I'm not sure which is correct.